presto-tests/src/test/java/com/facebook/presto/tests/tpch/TpchQueryRunner.java

```diff
@@ -51,23 +51,53 @@ public static DistributedQueryRunner createQueryRunner(Map<String, String> extra
         return queryRunner;
     }
 
-    public static DistributedQueryRunner createQueryRunner(Map<String, String> resourceManagerProperties, Map<String, String> coordinatorProperties, Map<String, String> extraProperties, int coordinatorCount)
+    public static DistributedQueryRunner createQueryRunner(Map<String, String> extraProperties, int coordinatorCount, boolean catalogServerEnabled)
```

Comment:
Are we using this method in any place?

Reply:
I originally had the catalogServer set on the createQueryRunner methods being used, but as @zacw7 recommended, I decided to remove them and leave it as is and introduce a method that would let the user, tester decide if they want to use this to enable the catalog server or not. Which is why this method is left for them to make use of or not

Reply:
Sure, will remove for now then